lib/matplotlib/tests/test_collections.py

```diff
@@ -626,6 +626,64 @@ def test_lslw_bcast():
     assert_equal(col.get_linewidths(), [1, 2, 3])
 
 
+@pytest.mark.style('default')
+def test_capstyle():
+    col = mcollections.PathCollection([], capstyle='round')
+    assert_equal(col.get_capstyle(), 'round')
+    col.set_capstyle('butt')
+    assert_equal(col.get_capstyle(), 'butt')
+
+
+@image_comparison(baseline_images=['capstyle'],
+                  extensions=['png'])
+def test_capstyle_image():
```

Comment:
Can you join the two image comparisons into a single test and thus a single reference image?  Reference test images are a large part of why the repo is so heavy, so we're trying to be careful with adding more of them.
After doing so, you can squash the old images out of the history with rebase; if you're not comfortable with it I can take care of it when merging too, just let me know.